src/main/resources/db/changelog/changes/04-create-users-roles-table.yaml

```diff
@@ -13,10 +13,14 @@ databaseChangeLog:
                     nullable: false
                     foreignKeyName: fk_user_id
                     references: users(id)
+                    primaryKey: true
+                    primaryKeyName: pk_users_roles

```

Comment:
Better to avoid changing the scripts that were already merged to main branch, because if we do such a thing our colleagues and staging environments will have an error that the script was changed